pkg/util/iswimi/iswimi.go

```diff
@@ -0,0 +1,14 @@
+package iswimi
```

Comment:
How about just `wimi` ?
`iswimi` is too specific and I'd hesitate to add new functions in this util package.